### PR TITLE
KVM: Use real DNS names for Kube-API endpoint

### PR DESCRIPTION
**Pull request by kiall**
This allows for valid and working certificates when using multi-master
deployments.

### CODE REVIEW

**Comment by kiall** [comment]
> ./tools/generate_environment.sh 3
	...
	> grep kubernetesHost environment.json 
	  "kubernetesHost": "kube-api-x3.devenv.caasp.suse.net",

	> ./tools/generate_environment.sh 2
	...
	> grep kubernetesHost environment.json 
	  "kubernetesHost": "kube-api-x2.devenv.caasp.suse.net",

	> dig +short kube-api-x3.devenv.caasp.suse.net
	10.17.2.1
	10.17.2.0
	10.17.2.2

	> dig +short kube-api-x2.devenv.caasp.suse.net
	10.17.2.0
	10.17.2.1

	> dig +short kube-api-x1.devenv.caasp.suse.net
	10.17.2.0

File: caasp-kvm/caasp-kvm [hunk 1]
```diff
@@ -113,7 +113,7 @@ build() {
     -var "kubic_velum_dir=$(realpath ../../velum)" \
     -var "kubic_caasp_container_manifests_dir=$(realpath ../../caasp-container-manifests)"
 
-  ./tools/generate_environment.sh
+  ./tools/generate_environment.sh $MASTERS
```
Comment (grahamhayes) [inline caasp-kvm/caasp-kvm]: can we not count the masters instead of adding inputs to a file that has none currently?

**Comment by grahamhayes** [comment]
One thing is that this now ties us to the R&D VPN to resolve the DNS entry. Are we OK with this?

**Comment by kiall** [comment]
> One thing is that this now ties us to the R&D VPN to resolve the DNS entry. Are we OK with this?

We already do that for the images, so, I think it's OK until we really fix those across the board

**Pull request merged**